### ISSUE TITLE
Any plugin settings influencing the SVG structure?

### ISSUE THREAD

**Issue by montrealist** [TomByrne/Flash2Svg#24]
First of all, thank you for the terrific work on this plugin! It's an enormous help in the workflow.

Question: are there any plugin settings that control the resulting SVG's **structure**? I have an animated FLA that exports fine - except it doesn't show up in IE Edge. I've got another SVG for reference (exported from a similar FLA), which is working fine. Their respective structures are quite different; I'm trying to reverse-engineer an export procedure which would give me an SVG structure similar to that of the working reference.

A couple of things that _could_ be causing issues:
- presence of the <animateTransform> element
- multiple <path> elements inside each <g> element

If there's a setting that influences either of the above - I would be thrilled to find out which one. Any pointers on where else I could dig are greatly appreciated!

Approximate structure of the problematic SVG:

```
<g id="Tween_33" transform="matrix(1 0 0 1 613.1 640.9)">
    <animateTransform attributeName="transform" additive="replace" type="translate" repeatCount="indefinite" dur="1.917s" keyTimes="..." calcMode="discrete" />
    <path fill="url(#linearGradient6)" d="..." />
    <path fill="#FCF0D4" d="..." />
    <path fill="#F8DC98" d="..." />
    <path fill="#4C4E50" d="..." />
    <path fill="#FFE83F" d="..." />
    <path fill="#FFC71E" d="..." />
</g>
```

Approximate structure of the working SVG:

```
<g id="4_RCAEID70187_144A.ai_f46_d70_l1" transform="matrix(1.02 0 0 1.02 -1437.75 -627.65)">
  <use xlink:href="#Whiteboard" transform="matrix(1 0 0 1 1489.35 687.95)" />
  <g mask="url(#4_RCAEID70187_144A.ai_f46_d70_l1_5_46)">
    <use xlink:href="#Bee" transform="matrix(1 0 0 1 1588.4 730.35)" />
  </g>
  <mask id="4_RCAEID70187_144A.ai_f46_d70_l1_5_46">
    <g id="Bee_mask" transform="matrix(1 0 0 1 1489.35 687.95)">
      <path fill="#FFF" d="..." fill-opacity="1" />
    </g>
  </mask>
</g>
```

**Comment by montrealist** [comment]
I may have found a workaround, using "before/after anim" setting, to show the last frame of the animation as fallback - this works in Edge. 

However, the setting appears to be absent from the UI in versions after 3.32. In the source code it seems the [relevant HTML element](https://github.com/TomByrne/Flash2Svg/commit/947531dad483471608e7ba9e21a49a0ee4c22c5f#diff-4464bd0625c2045030a3e520522b600fR259) was already commented out on commit - any pointers?

**Comment by montrealist** [comment]
One more follow-up question: is it possible to invoke the plugin from the command line in any way?

**Comment by TomByrne** [comment]
Hi Max,
Firstly, no it's not possible to use the plugin from the command line as all of the code is written for the built-in JS Engine in Animate/Flash. Also, as it relies on E4X, and depends on the Animate DOM it isn't so easy to port to another JS system.

Originally, I couldn't find a way to display a different state before the animation started and after it finished, so I gave the user the option to select whether they would look like the start or the end of the animation. About a year ago I found a way to display different states before the animation started and after it concluded, and in the interest of simplifying an already very complex codebase, I chose to remove the option from the UI.

To be clear, this means that before an animation starts, it will look like the first frame, and after it finishes, it will look like the last frame. I think this is the most intuitive behaviour.
Also, this means that in IE/Edge the SVG will look like the first frame.
